### PR TITLE
adding draft for importing via CLI

### PR DESCRIPTION
**Pull request by nathanweatherly**
https://github.com/open-cluster-management/backlog/issues/16088

### CODE REVIEW

**Review by zkayyali812 — APPROVED** [review]

File: import_cli.adoc [hunk 1]
```diff
@@ -0,0 +1,183 @@
+[#importing-a-managed-cluster-with-the-cli]
+= Importing a managed cluster with the CLI
```
Comment (joeg-pro) [inline import_cli.adoc:2]: Since MCE has no UI, I think the "with the CLI" qualification here is unnecessary as we don't have to contrast with another section that would be "with the UI".  Also, strictly speaking, you don't import a managed cluster because the cluster isn't considered managed until the import is done.  So maybe just "Importing a cluster" or "Importing a cluster into the inventory" might work better?  (While "Importing a cluster to bring it under management" might be the most accurate, its seems too wordy).

File: import_cli.adoc [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+After you install multicluster engine for Kubernetes, you are ready to import a cluster to manage.
```
Comment (joeg-pro) [inline import_cli.adoc:4]: Sug:  "...import a cluster to manage." -> "import a cluster so that you can manage it with the engine."

(I think its ok to refer to an installed instance of MCE simply as "the engine" vs. always repeating the MCE name.)

File: import_cli.adoc [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+Using the OpenShift Container Platform CLI, you can import a cluster using the `kubeconfig` file of the managed cluster. Alternatively, you can run import commands manually on the managed cluster.
```
Comment (joeg-pro) [inline import_cli.adoc:5]: Sug: "...of the managed clsuter." -> "of the cluster to be managed."  (2x) (This is more of the idea that the cluster isn't really a managed cluster until *after* its imported.)

**Review by joeg-pro — CHANGES_REQUESTED** [review]
See individual comments.

File: import_cli.adoc (hunk 2 above)
Comment (nathanweatherly) [reply import_cli.adoc:4]: I've discussed this with @swopebe-- my understanding is `the engine` or `the engine cluster` isn't currently acceptable (I had "the engine cluster" a bunch and started using the full name). 

For now let's use the full name. If @swopebe agrees there's a shorter name that we can agree to use after we say "multicluster engine for Kubernetes" the first time, then slimming it down should be easy.

File: import_cli.adoc (hunk 1 above)
Comment (nathanweatherly) [reply import_cli.adoc:2]: Yeah, i struggle to find a useful phrase here. Maybe "import cluster" since it's not managed yet?

File: import_cli.adoc (hunk 3 above)
Comment (nathanweatherly) [reply import_cli.adoc:5]: Maybe "cluster for import" or "cluster to import"?

Comment (nathanweatherly) [reply import_cli.adoc:5]: Or "import cluster" ?

**Review by cameronmwall — APPROVED** [review]

**Comment by nathanweatherly** [comment]
messed up git history, redoing pr
